src/javascripts/modules/pillbar.js

```diff
@@ -0,0 +1,28 @@
+export default class Pillbar {
+  constructor(el) {
+    this.el = el
+    this.ticking = false
+    this.lastScrollY = 0
+
+    window.addEventListener('scroll', () => this.handleScroll())
+  }
+
+  handleScroll() {
+    this.lastScrollY = window.scrollY
+    if (!this.ticking) {
+      window.requestAnimationFrame(() => {
+        this.togglePillbar()
+      })
+    }
+    this.ticking = true
+  }
+
+  togglePillbar() {
+    this.ticking = false
```

Comment:
what does ticking mean? maybe it means is the user scrolling, in which case `isUserScrolling` may be a better named variable.

Reply:
@benjtinsley I pointed Peyton towards some rAF articles where ticking is used to prevent the calculation from happening too much. e.g.: https://developer.mozilla.org/en-US/docs/Web/Events/scroll